src/probnum/linalg/solvers/_probabilistic_linear_solver.py

```diff
@@ -63,7 +65,8 @@ class ProbabilisticLinearSolver(
     See Also
     --------
     ~probnum.linalg.problinsolve : Solve linear systems in a Bayesian framework.
-    ~probnum.linalg.bayescg : Solve linear systems with prior information on the solution.
+    ~probnum.linalg.bayescg : Solve linear systems with prior information on the
+    solution.
```

Comment:
```suggestion
    ~probnum.linalg.bayescg : Solve linear systems with prior information on the solution.
```
A line break here causes the built docs to display another link instead of the full sentence as a description of `probnum.linalg.bayescg`.
![Screenshot from 2022-04-08 18-32-29](https://user-images.githubusercontent.com/29815984/162542520-dfc10188-c0b6-4225-9c9a-899b1f1b06bc.png)